src/server/project.ts

```diff
@@ -715,6 +712,26 @@ namespace ts.server {
             }
         })();
 
+        private _isJsInferredProject = false;
+        set isJsInferredProject(newValue: boolean) {

```

Comment:
personal nit: I'm not a big fan of set-only accessors, can you turn it into method instead?